### PR TITLE
Feature/list view pages

### PR DESCRIPTION
**Pull request by frozenhelium**

### CODE REVIEW

**Review by priyesh777 — APPROVED** [review]

**Review by priyesh777 — APPROVED** [review]

**Review by priyesh777 — APPROVED** [review]

File: src/components/Tooltip/index.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        () => {
+            const handleMouseEnter = () => {
+                setShowPopup(true);
+            };
+
+            const handleMouseOut = () => {
+                setShowPopup(false);
+            };
+
+            if (dummyRef.current) {
```
Comment (tnagorra) [inline src/components/Tooltip/index.tsx]: Let's return early. Also, let's not use parentRef.current on cleanup

File: src/components/Tooltip/index.tsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        },
+        [],
+    );
+
+    return (
+        <>
+            {!hasParentRef && (
+                <div
+                    ref={dummyRef}
+                />
```
Comment (tnagorra) [inline src/components/Tooltip/index.tsx]: Let's style this as fixed element with visibility hidden

File: src/views/AllFieldReports/index.tsx [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+                    renderer: (country) => (
+                        <Link
+                            to={generatePath(
+                                countryRoute.absolutePath,
+                                { countryId: String(country.id) },
+                            )}
+                        >
+                            {country.name}
+                        </Link>
+                    ),
```
Comment (tnagorra) [inline src/views/AllFieldReports/index.tsx]: These 2 should be memoized.

File: src/views/AllFlashUpdates/index.tsx [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+            renderer: (countryDistrict: CountryDistrict) => (
+                <Link
+                    to={generatePath(
+                        countryRoute.absolutePath,
+                        { countryId: String(countryDistrict.country) },
+                    )}
+                >
+                    {countryDistrict.country_details.name}
+                </Link>
+            ),
```
Comment (tnagorra) [inline src/views/AllFlashUpdates/index.tsx]: These functions should be separately memoized

File: src/views/Home/HighlightedOperations/OperationCard/index.tsx [hunk 5]
```diff
@@ -78,15 +83,42 @@ function OperationCard(props: Props) {
...
     const isSubscribed = subscriptionMap[id] ?? false;
+    let countriesInfoDisplay = strings.operationCardNoCountryInvolved;
+
+    if (countries.length > 0) {
+        if (countries.length === 1) {
+            countriesInfoDisplay = countries[0].name;
+        } else {
+            countriesInfoDisplay = strings.operationCardInvolvesMultipleCountries;
+        }
+    }
```
Comment (tnagorra) [inline src/views/Home/HighlightedOperations/OperationCard/index.tsx]: Let's simplify

**Review by tnagorra — CHANGES_REQUESTED** [review]

**Review by AdityaKhatri — APPROVED** [review]

**Pull request merged**